src/arrow1/writer_properties.rs

```diff
@@ -22,10 +22,10 @@ impl From<Compression> for parquet::basic::Compression {
         match x {
             Compression::UNCOMPRESSED => parquet::basic::Compression::UNCOMPRESSED,
             Compression::SNAPPY => parquet::basic::Compression::SNAPPY,
-            Compression::GZIP => parquet::basic::Compression::GZIP,
-            Compression::BROTLI => parquet::basic::Compression::BROTLI,
+            Compression::GZIP => parquet::basic::Compression::GZIP(Default::default()),
```

Comment:
ah TIL you can pass `Default::default()` instead of manually using `GzipLevel::default()`

Reply:
Keep this change when you rebase 😄